.circleci/config.yml

```diff
@@ -67,8 +67,9 @@ workflows:
             .circleci/.* validate-bqetl true
             .circleci/.* validate-routines true
   setup-tag:
-    when:
-      matches: {pattern: '^[0-9]{4}.[0-9]{1,2}.[0-9]+$', value: << pipeline.git.tag >>}
     jobs:
       - continuation/continue:
           configuration_path: .circleci/workflows.yml
+          filters:
+            branches:
+              ignore: /.*/
```

Comment:
This seems to be working when I just tested it. Filtering on tags does however not work as part of `continuation`, but that should be handled in the pypi workflow